server/pulp/plugins/util/publish_step.py

```diff
@@ -577,9 +602,11 @@ def get_distributor_type(self):
 
     def publish(self):
         """
+        DEPRECATED
```

Comment:
For this and other deprecated methods, is there a reason not to warnings.warn a DeprecationWarning?